bigcode_eval/utils.py

```diff
@@ -341,6 +354,23 @@ def complete_code(
                 # reset gen_token_dict - prevent redundant decoding
                 gen_token_dict = defaultdict(list)
 
+    if raw_generation_path:
+        raw_code_gens = copy.deepcopy(code_gens)
+        raw_code_gens = update_code_gens(
+            task,
+            tokenizer,
+            limit_start,
+            prefix,
+            instruction_tokens,
+            False,
```

Comment:
Please use a kwarg `postprocess=False` here so it's clear what's being set to false